default_export.js

```diff
@@ -0,0 +1,7 @@
+export default class Animal {
+  constructor(name) {
+    this.name = name;
+  }
+};
+let animal = new Animal("fox");

```

Comment:
you should not use your class in the same file -- you should import it in the main file and use it there